spec/bundler/shared_helpers_spec.rb

```diff
@@ -209,6 +209,10 @@
   end
 
   describe "#set_bundle_environment" do
+    before do
+      ENV["BUNDLE_GEMFILE"] = "Gemfile"
```

Comment:
Isn't looking for a file called `Gemfile` the default? What's the impetus behind this setup?